backend/db/repository/posts_repo.py

```
@@ -0,0 +1,89 @@
+    
+def user_liked_post(user_id:str, post_id:str):
+    """
+    Docstring for user_liked_post
+    
+    :param user_id: Description
+    :type user_id: str
+    :param post_id: Description
+    :type post_id: str
+    """
+    try:
+        res = supabase.table("user_liked_posts").insert({"userid":user_id, "likedpostid":post_id}).execute()
```

Ruff (0.14.8)

Local variable `res` is assigned to but never used

Remove assignment to unused variable `res`

(F841)